components/style/properties/longhand/svg.mako.rs

```diff
@@ -105,10 +105,13 @@ ${helpers.single_keyword("mask-repeat",
 
 <%helpers:vector_longhand name="mask-position-x" products="gecko" animatable="True" extra_prefixes="webkit"
                           spec="https://drafts.fxtf.org/css-masking/#propdef-mask-position">
-    use std::fmt;
-    use style_traits::ToCss;
-    use values::HasViewportPercentage;
-    use values::specified::position::HorizontalPosition;
+    use properties::animated_properties::{Interpolate, RepeatableListInterpolate};
+    use properties::longhands::mask_position_x::computed_value::T as MaskPositionX;
+    pub use properties::longhands::background_position_x::single_value::get_initial_value;
+    pub use properties::longhands::background_position_x::single_value::get_initial_position_value;
+    pub use properties::longhands::background_position_x::single_value::get_initial_specified_value;
+    pub use properties::longhands::background_position_x::single_value::parse;
+    pub use properties::longhands::background_position_x::single_value::SpecifiedValue;
 
     pub mod computed_value {
```

Comment:
I don't think so. Since the implementation of  `competed_value` is different between `mask_position_x` and `background_position_x`.  Trait `RepeatableListInterpolate` and `Interpolate` are required to implemented in the `computed_value` in `mask_position_x`